esx_service/cli/vmdkops_admin.py

```diff
@@ -1225,20 +1232,12 @@ def config_init(args):
         except auth_data.DbAccessError as ex:
             return err_out("Fatal: DB link or DB is corrupted. Check {} ({}".format(link_path, ex))
 
-    if auth.mode  == auth_data.DBMode.NotConfigured:
+    if auth.mode == auth_data.DBMode.NotConfigured:
```

Comment:
It would be nice to have separate method for state transition which internally validates current state against new state.